src/rgw/rgw_admin.cc

```diff
@@ -2998,7 +2998,7 @@ int main(int argc, const char **argv)
           if (!conn) {
             cerr << "failed to find a zone or zonegroup for remote "
                 << remote << std::endl;
-            return -ENOENT;
```

Comment:
you might want to fill the commit message with the rationale behind this change? for example, why shall we have positive return codes instead of negative ones?

and could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes. please note, the title of PR is not that of commit message.

Reply:
agree with kefu